.github/CODEOWNERS

```diff
@@ -6,32 +6,13 @@
 ################
 # As of 1/30/2023 these paths have no owners:
 
-# /
+# /**
```

Comment:
Do we ever want this catchall path in our CODEOWNERS files?

Reply:
I listed it like that to bring to the attention the fact there are bunch of files in the root dir that have no owners. Didn't want to enumerate all of them.